### ISSUE TITLE
Failure to Install

### ISSUE THREAD

**Issue by BKDaugherty** [gadomski/stac-incubator-rs#4]
Heyo! I realize this is an incubator project, but thought I'd report my failure anyways. 

I tried to install `stac-cli` but got a failure.

```
❯ rustup show
Default host: x86_64-unknown-linux-gnu
rustup home:  /home/brendon/.rustup

installed toolchains
--------------------

stable-x86_64-unknown-linux-gnu (default)
nightly-x86_64-unknown-linux-gnu

installed targets for active toolchain
--------------------------------------

wasm32-unknown-unknown
x86_64-unknown-linux-gnu

active toolchain
----------------

stable-x86_64-unknown-linux-gnu (default)
rustc 1.63.0 (4b91a6ea7 2022-08-08)

```

```
$ cargo install stac-cli
...
   Compiling stac-cli v0.0.1
error[E0282]: type annotations needed
  --> /home/brendon/.cargo/registry/src/github.com-1ecc6299db9ec823/stac-cli-0.0.1/src/download.rs:83:31
   |
83 |             .with_key("eta", |state| format!("{:.1}s", state.eta().as_secs_f64()))
   |                               ^^^^^
   |
   = note: type must be known at this point
help: consider giving this closure parameter an explicit type
   |
83 |             .with_key("eta", |state: _| format!("{:.1}s", state.eta().as_secs_f64()))
   |                                    +++

For more information about this error, try `rustc --explain E0282`.
error: could not compile `stac-cli` due to previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `stac-cli v0.0.1`, intermediate artifacts can be found at `/tmp/cargo-installwSqExA`
```

**Comment by BKDaugherty** [comment]
It doesn't look like we use the "eta" key? And [it looks like that key should be built in.](https://docs.rs/indicatif/latest/indicatif/index.html#templates) Can I just delete that line?

**Comment by gadomski** [comment]
Fixed in #6 and updated to v0.0.2 on crates.io: https://crates.io/crates/stac-cli. I was able to install successfully on my machine, so I'm closing. Please re-open if the issue isn't fixed for you. Thanks for the report!
